### PR TITLE
VTCCA-3206: implemented assign/unassign agent to/from some properties

### PR DESCRIPTION
**Pull request by habeeb01**

### CODE REVIEW

File: app/uk/gov/hmrc/voapropertylinking/models/modernised/agentrepresentation/AssignAgent.scala [hunk 1]
```diff
@@ -18,8 +18,8 @@ package uk.gov.hmrc.voapropertylinking.models.modernised.agentrepresentation
 
 import play.api.libs.json.{Json, OFormat}
 
-case class AppointAgent(agentRepresentativeCode: Long, scope: String)
```
Comment (DanielConnelly) [inline app/uk/gov/hmrc/voapropertylinking/models/modernised/agentrepresentation/AssignAgent.scala]: I swear you rename this file every PR.

**Comment by DanielConnelly** [comment]
LGTM 👍

File: app/uk/gov/hmrc/voapropertylinking/models/modernised/agentrepresentation/AssignAgent.scala (hunk 1 above)
Comment (habeeb01) [reply app/uk/gov/hmrc/voapropertylinking/models/modernised/agentrepresentation/AssignAgent.scala]: nope. This is the very first time. I renamed something with a similar name in the frontend. Maybe thats what you're thinking of

**Comment by reddy-yattapu** [comment]
:+1:

**Pull request merged**